requirements.txt

```diff
@@ -1,14 +1,11 @@
-django >= 3.2, <3.3
-xlwt == 1.3.0
-xlutils == 2.0.0
-openpyxl == 3.0.9
-psycopg2-binary == 2.9.3
-redis == 4.1.1
-django-redis == 5.2.0
-django-fsm == 2.8.0
-django-webtest == 1.9.9
-WebTest == 3.0.0
-model-bakery == 1.4.0
-django-extensions == 3.1.5
-django-sendfile2 == 0.6.0
-mozilla-django-oidc == 1.2.4
+django-extensions==3.1.5
+django-fsm==2.8.0
+django-redis==5.2.0
+django-sendfile2==0.6.1
+django>=3.2,<3.3
+mozilla-django-oidc==2.0.0
```

Comment:
From what I can tell, the only breaking changes really only are that they dropped support for python 2 and old 3 versions, right?